packages/medusa/src/api-v2/admin/invites/accept/route.ts

```diff
@@ -19,11 +20,18 @@ export const POST = async (req: MedusaRequest, res: MedusaResponse) => {
 
   const input = {
     invite_token: req.filterableFields.token as string,
-    auth_user_id: req.auth_user!.id,
+    auth_user_id: req.auth?.auth_user_id,
     user: req.validatedBody as AdminPostInvitesInviteAcceptReq,
   } as InviteWorkflow.AcceptInviteWorkflowInputDTO
 
-  const { result: users } = await workflow.run({ input })
+  let users
+  try {
+    const { result } = await workflow.run({ input })
+    users = result
+  } catch (e) {
+    res.status(401).json({ message: "Unauthorized" })
```

Comment:
suggestion: we should probably log something here, the reason being that it can fail for many reason other than unauthorized and the log wouldn't show that information and make difficult to debug. wdyt?